client/src/components/navHeader.vue

```diff
@@ -46,6 +46,8 @@
                 placeholder="Search"
                 @keydown.enter.exact.prevent="onSubmitNavbar"
               ></b-form-input>
+              <b-form-checkbox v-model="textMatchAllButton"></b-form-checkbox>
```

Comment:
Title? with a javascript to say "OR" when not checked, and "AND" where checked, with a title tip describing a bit more in detail 'Selecting AND will require all provided terms to be in the result"

Reply:
added. see if you can see the tooltip on your end. I can see it may 20% of the time.